### PR TITLE
New App: iRedMail - Open Source Mail Server

### PR DESCRIPTION
**Pull request by ido-pluto**
#### This is a custom build of iRedMail.

I added some fixes to make this work with CapRover + Some adjustment I think is needed.

Changes:
- Remove the example certificate from the web app so it works with CapRover HTTPS
- Added link to the Webmail from the iRedMail Admin
- Added DNS records in the 'Edit domain' page
- Option to change the installed version of iRedMail Admin (On the first build, not work after docker image built)

### CODE REVIEW

**Comment by githubsaturn** [comment]
This is very customized and very complex. I don't think it's maintainable in its current form. I suggest either make it a tad more maintainable my moving the changes upstream (to the source code) or remove the customizations.

**Comment by idoio** [comment]
True, but that way, my docker file will work with every update of the original docker image without the need to do anything :)

btw: if the build doesn't work after formatting with prettier, I wilt upload it on JSON format

**Comment by githubsaturn** [comment]
> but that way, my docker file will work with every update of the original docker image without the need to do anything :)

Not sure if I understand. Are you saying the current method works with every update moving forward?

Let me rephrase my question. Why wouldn't you want to move these changes to the main repo? Why do we need to have these customization in CapRover one click app, but not the default (non-CapRover) installation?

**Comment by idoio** [comment]
Yes - spouse too. I don't move the changes because there are some changes that make iRedMail compatible with CapRover.
If you install without CapRover, you don't need those.

I guess that all of my changes are necessary because in 'one-click-app' you are not expected to go to the shell.

For example, the "DNS records" changes are compatible only with ubuntu, so there is no point to push them to the main repo.
I am not trying to change iRedMail, just trying to make it compatible and comfortable to use with CapRover.

**Comment by githubsaturn** [comment]
I see. Okay, let's do this.


Regarding formatting, feel free to revert to yaml. I'll merge and then will sort out the prettier issue.
